setup.py

```diff
@@ -29,7 +29,7 @@
 # Python Package Setup
 setup(
     name='cufinufft',
-    version='1.2',
+    version='1.3',
     author='Yu-shuan Melody Shih, Garrett Wright, Joakim Anden, Johannes Blaschke, Alex Barnett',
     author_email='yoyoshih13@gmail.com',
```

Comment:
How about your email here, Joakim? :)

Reply:
Sure why not.

Reply:
Done (see 1472cb8c96fd7dba21df25336ba292cf07cbbb6a).